output.md

```diff
@@ -97,9 +97,9 @@ file.h5
     ...
     snapshot_n/
 ```
-
+In addition to that, psydac also has a class to read those files, recreate and the `FemSpace` and `FemField` objects and export them to `VTK`. 
 ## Usage
-The `OutputManager` class is meant to be used as follows:
+Those two classes are meant to be used as follows. The `#` line is supposed to represent the end of a file.
```

Comment:
I think it is much clearer to show two separate code snippets, rather than saying "The `#` line is supposed to represent the end of a file."

Reply:
Fixed in 9d1e1b5

Reply:
Thanks. I have just expanded the explanations